tsdb/db.go

```diff
@@ -2280,10 +2280,8 @@ func (db *DB) CleanTombstones() (err error) {
 					db.logger.Error("failed to delete block after failed `CleanTombstones`", "dir", dir, "err", err)
 				}
 			}
-			if err != nil {
-				return fmt.Errorf("reload blocks: %w", err)
-			}
-			return nil
+
+			return fmt.Errorf("reload blocks: %w", err)
```

Comment:
```suggestion
                       // This should only be reached if an error occurred.
			return fmt.Errorf("reload blocks: %w", err)
```
maybe we can add a comment, this one is tricky